requirements.txt

```diff
@@ -1,5 +1,6 @@
 BeautifulSoup==3.2.0
```

Comment:
These probably need to be removed...

Reply:
I think is used in other places, like messages/views.py